Testscripts/Windows/WORKLOAD-HB.ps1

```diff
@@ -166,37 +235,18 @@ echo disk > /sys/power/state
 		$TestProvider.RestartAllDeployments($AllVMData)
 
 		# Check the VM status before hibernation
-		$vmStatus = Get-AzVM -Name $vmName -ResourceGroupName $rgName -Status
+		$vmStatus = Get-AzVM -Name $AllVMData[0].RoleName -ResourceGroupName $rgName -Status
 		if ($vmStatus.Statuses[1].DisplayStatus -eq "VM running") {
-			Write-LogInfo "$($vmStatus.Statuses[1].DisplayStatus): Verified successfully VM status is running before hibernation"
+			Write-LogInfo "$($vmStatus.Statuses[1].DisplayStatus): Verified successfully VM $($AllVMData[0].RoleName) status is running before hibernation"
 		} else {
-			Write-LogErr "$($vmStatus.Statuses[1].DisplayStatus): Could not find the VM status before hibernation"
-			throw "Can not identify VM status before hibernate"
+			throw "Can not proceed because VM $($AllVMData[0].RoleName) status was $($vmStatus.Statuses[1].DisplayStatus)"
 		}
 
-		# Send fio 32-job command for 5 min
-		Write-LogInfo "Running fio-1 command"
-		$timeout = New-Timespan -Minutes $maxFIORunWaitMin
-		$sw = [diagnostics.stopwatch]::StartNew()
-		$state = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "bash ./fio1Command.sh" -RunInBackground -runAsSudo
-		Wait-Time -seconds 5
-		while ($sw.elapsed -lt $timeout){
-			Wait-Time -seconds 15
-			$state = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password "cat /home/$user/state.txt"
-			if ($state -eq "TestCompleted") {
-				Write-LogInfo "Completed fio command execution in the VM $($AllVMData.RoleName) successfully"
-				break
-			} else {
-				Write-LogInfo "$($state): fio command is still running!"
-			}
-		}
-		if ($state -ne "TestCompleted") {
-			throw "fio-1 command is still running after $maxFIORunWaitMin minutes"
-		}
+		WorkLoad 1
```

Comment:
change function name, it is hard to understand this line

Reply:
Start-Workload is ok?